src/2023/index.html

```diff
@@ -8,41 +8,92 @@
     <link rel="stylesheet" href="styles/styles.css">
```

Comment:
Please note that I have added a new `<link>` tag for Mastadon in #15

Reply:
Thank you, I will rebase.